modules/cpmso-feat-auto-update.adoc

```diff
@@ -2,12 +2,49 @@
 //
 // * machine_management/control_plane_machine_management/cpmso-managing-machines.adoc
 
+ifeval::["{context}" == "control-plane"]
+:rosa-classic:
+:osd:
+endif::[]
+
 :_mod-docs-content-type: CONCEPT
 [id="cpmso-feat-auto-update_{context}"]
 = Automatic updates to the control plane configuration
+//Not for ROSA/OSD:
+ifndef::openshift-dedicated,openshift-rosa[]
+The `RollingUpdate` update strategy automatically propagates changes to your control plane configuration.
+This update strategy is the default configuration for the control plane machine set.
+endif::openshift-dedicated,openshift-rosa[]
+//For ROSA:
+ifdef::openshift-rosa[]
+On ROSA (classic architecture) clusters, control plane machine sets automatically propagate changes to your control plane configuration.
+The Control Plane Machine Set Operator creates a replacement control plane machine with the configuration that is specified in the `ControlPlaneMachineSet` CR. The Operator replaces control plane machines in a way that mitigates any potential negative effects on cluster API or workload availability.
+endif::openshift-rosa[]
+//For OSD:
+ifdef::openshift-dedicated[]
+On {product-title} clusters, the control plane machine sets automatically propagate changes to your control plane configuration.
+The Control Plane Machine Set Operator creates a replacement control plane machine with the configuration that is specified in the `ControlPlaneMachineSet` CR. The Operator replaces control plane machines in a way that mitigates any potential negative effects on cluster API or workload availability.
+endif::openshift-dedicated[]
+//Not for ROSA/OSD:
+ifndef::openshift-dedicated,openshift-rosa[]
+
+For clusters that use the `RollingUpdate` update strategy, the Operator creates a replacement control plane machine with the configuration that is specified in the CR.
+endif::openshift-dedicated,openshift-rosa[]
+//Anything not enclosed in an ifdef or ifndef prints for everyone:
+
+When the replacement control plane machine is ready, the Operator deletes the control plane machine that is marked for replacement because the machine is either out of spec or in an unhealthy state.
+The replacement machine then joins the control plane.
```

Comment:
```suggestion
A control plane machine may be marked for replacement at any time, typically because the machine has fallen out of spec or entered an unhealthy state. Such replacements are a normal part of a cluster's lifecycle and not a cause for concern. Should any part of a control plane node replacement fail, SRE will be alerted to the issue automatically.
```
I'm trying to highlight to customers that they need not be worried (nor open a support ticket) when they see these nodes get replaced.
```